providers/ionoscloud/server.go

```diff
@@ -48,10 +48,23 @@ func (g *ServerGenerator) InitResources() error {
 				continue
 			}
 
+			var resourceType string
+
+			if *server.Properties.Type == "ENTERPRISE" {
+				resourceType = "ionoscloud_server"
+			} else if *server.Properties.Type == "CUBE" {
+				resourceType = "ionoscloud_cube_server"
+			} else if *server.Properties.Type == "VCPU" {
+				resourceType = "ionoscloud_vcpu_server"
+			} else {
+				log.Printf("[WARNING] unknown server type: %v for server with ID: %v, skipping this server", *server.Properties.Type, *server.Id)
+				continue
+			}
+
```

Comment:
Can be a small function GetServerResourceType with a switch

Reply:
@cristiGuranIonos done